formspree/settings.py

```diff
@@ -22,6 +22,7 @@
 SERVICE_NAME = os.getenv('SERVICE_NAME') or 'Forms'
 UPGRADED_PLAN_NAME = os.getenv('UPGRADED_PLAN_NAME') or 'Gold'
 SERVICE_URL = os.getenv('SERVICE_URL') or 'http://example.com'
+SERVICE_UPGRADE = os.getenv('SERVICE_UPGRADE') or 'https://formspree.io/account'
```

Comment:
Do you think this is absolutely needed? I'm kinda against having so many settings. We already have so much! In the landing page and in the emails we send after the user reaches 1000 submissions I think there's a `{{ url_for('account') }}` hardcoded. If we are going to change this here we should change it there also (but I personally prefer the `url_for` approach).